providers/redfish/tasks.go

```diff
@@ -52,27 +36,6 @@ func (c *Conn) GetFirmwareInstallTaskQueued(ctx context.Context, component strin
 	return task, nil
 }
 
-// purgeQueuedFirmwareInstallTask removes any existing queued firmware install task for the given component slug
-func (c *Conn) purgeQueuedFirmwareInstallTask(ctx context.Context, component string) error {
```

Comment:
This method is required - https://github.com/bmc-toolbox/bmclib/blob/f096534cec29f6fc4c415a698bd883bdcf5b4894/providers/redfish/firmware.go#L65